### DISCUSSION TITLE
What is the plugins directory used for?

### DISCUSSION THREAD

**Discussion by ruanwenjun** [apache/seatunnel#1606] [Q&A] [answered]
I find there are a directory `plugins/json/files/schemas` in seatunnel, and find no document about this directory.

Is this directory needed? If needed, we may need to add some description about this. Either, we should delete.

**Comment by ruanwenjun** [comment, marked as answer]
@BenJFan @wuchunfu @CalvinKirs

**Reply by Hisoka-X** [reply]
I don't know why too

**Reply by ruanwenjun** [reply]
> 

I decide to delete this directory.

**Reply by CalvinKirs** [reply]
Before is a simple configuration file example

**Reply by ruanwenjun** [reply]
> Before is a simple configuration file example

Ok, let's delete this directory, since we have a config directory to show the example.

**Comment by ruanwenjun** [comment]
The `plugins` directory is used as a default schema dir in json transform plugin. https://seatunnel.apache.org/docs/transform/json#schema_dir-string
